std/zip.d

```diff
@@ -760,16 +760,22 @@ public:
 
             if (numberOfThisDisc < numberOfStartOfCentralDirectory) continue;
 
+            uint k = start - zip64EndOfCentralDirLocatorLength;
+            auto maybeZip64 = k < start && _data[k .. k + 4] == zip64EndOfCentralDirLocatorSignature;
```

Comment:
If `zip64EndOfCentralDirLocatorLength` is a non-zero constant, and here `k == start - zip64EndOfCentralDirLocatorLength`, when would `k < start` be false? If it underflows?

Reply:
It's basically a copy of line 619. And yes, I think it's to detect underflows.